apps/condo/domains/ticket/hooks/useTableColumns.tsx

```diff
@@ -113,9 +114,16 @@ export function useTableColumns <T> (filterMetas: Array<FiltersMeta<T>>) {
                         </Tag>
                     )
                 }
+                {
+                    record.isWarranty && (
+                        <Tag color='purple'>
```

Comment:
Better to move colors to apps/condo/domains/ticket/constants/style.js and give it a name.

Reply:
Done